File: 백준/Bronze/2420. 사파리월드/사파리월드.cs
using System;
using System.Numerics;

class Q2420
{
    static void Main(string[] args)
    {
        string[] S = Console.ReadLine().Split();
        long N = int.Parse(S[0]);
        long M = int.Parse(S[1]);
        
        Console.WriteLine(Math.Abs(N - M));
    }
}

File: 백준/Bronze/2420. 사파리월드/사파리월드.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        long N = sc.nextInt();
        long M = sc.nextInt();
        
        System.out.print(Math.abs(N - M));
    }
}